### PR TITLE
Bros 8 poker mvp continuation

### PR DESCRIPTION
**Pull request by gaugerard**

### CODE REVIEW

**Comment by bapierre** [comment]
2 code smells => not merging

File: analytics/src/test/java/brocoeur/example/analytics/service/ServiceRequestStatusServiceTest.java [hunk 1]
```diff
@@ -26,6 +26,7 @@
 import static brocoeur.example.common.OfflineGameStrategyTypes.OFFLINE_COIN_TOSS_RANDOM;
 import static brocoeur.example.common.ServiceRequestTypes.*;
 import static org.mockito.Mockito.*;
+import static org.mockito.Mockito.verify;
```
Comment (bapierre) [inline analytics/src/test/java/brocoeur/example/analytics/service/ServiceRequestStatusServiceTest.java:30]: Redundant no ?

**Review by bapierre — CHANGES_REQUESTED** [review]
Seems good to me (aka a noob) just change the Mockito redundancy

File: analytics/src/test/java/brocoeur/example/analytics/service/ServiceRequestStatusServiceTest.java (hunk 1 above)
Comment (gaugerard) [reply analytics/src/test/java/brocoeur/example/analytics/service/ServiceRequestStatusServiceTest.java:30]: yes

**Pull request merged**